docs/visualize.md

```diff
@@ -0,0 +1,99 @@
+# Installation
+
+To create state machine graphs you must to install `automat` with the graphing dependencies.
+
+```
+pip install automat[visualize]
+```
+
+# Example
+
+Given a simple project file structure of:
+
+```
+.
+└── mystatemachine
+    └── __init__.py
+```
+
+And the following state machine defined in `__init__.py`
+
+```
+from automat import MethodicalMachine
+
+class MyMachine(object):
+    _machine = MethodicalMachine()
+
+    @_machine.state(initial=True)
+    def state_a(self):
+        """
+        State A
+        """
+
+    @_machine.state()
+    def state_b(self):
+        """
+        State B
+        """
+
+    @_machine.input()
+    def change_state(self):
+        """
+        Change state
+        """
+
+    @_machine.input()
+    def output_on_change_state(self):
+        """
+        Change state
+        """
+        return "Changing state"
+
+    state_a.upon(change_state, enter=state_b, outputs=[output_on_change_state])
+```
+
+A state machine visualization can be generated with:
```

Comment:
Eliminate the passive voice:

"You can generate a state machine visualization by running:"